connectors/filesystem.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: File System
+description: "Write Parquet and JSON to S3 and local filesystems"
+---
+
+Arroyo provides the capability to write Parquet and JSON files to S3 and local filesystems.
+This will produce complete files in line with the checkpointing system.
```

Comment:
I think the work involved here is awesome enough that it warrants a few more lines about how Arroyo ensures transactional sinking to filesystems.